java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,317 @@
+.. Select the *Instance* on which the method is invoked.
+.. In *Class*, select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Method*, search and select the method to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
+include::connectors::partial$acb-java-module-partial.adoc[tag="advanced-tab-attributes"]
+
+[[configuring-invoke-static]]
+=== Configuring the Invoke Static Operation
+
+Invoke static methods with the arguments you provide. 
+
+To configure the attributes for *Invoke* in the Anypoint Code Builder UI:
+
+. Click the *Invoke Static* component on the canvas.
+. In the *General* tab:
+.. In *Class*, select the Java class that contains the method to use. 
++
+You can search for all the available Java classes for your project, including classes you create and native Java classes.  
+.. In *Method*, search and select the method to use. 
++
+NOTE: If this is blank, click the refresh icon to populate it.
+include::connectors::partial$acb-java-module-partial.adoc[tag="advanced-tab-attributes"]
+
+=== Configuring the Validate Type Operation
+
+Use this operation to validate that an instance is an `instanceof` a class.
+
+. Click the *Validate type* component on the canvas.
+. In the *General* tab:
+.. Select the *Class* that contains the method to use.
+.. Enter the name of the *Instance* on which the method is invoked.
+.. Select *Accept subtypes* to accept subtypes of a class. 
++
+Unselect this option if the instance has to be of the exact same class. 
+
+== Example
+
+[[creating-java-class]]
+=== Creating a Java Class
+
+Create Java classes containing methods to use in your application flows. The classes you create in your project are not available to other projects in your Anypoint Code Builder workspace.
+
+. In your Anypoint Code Builder project, create a new Java package under `src/main/java`.
+. Create a new Java class called `TaxCalculator.java`, and add the implementation, for example:
++
+[source,java,linenums]
+----
+package com.me;
+
+public class TaxCalculator {
+
+    private Double percentBaseTax = 7.0;
+
+    public Double calculateTax(Double price, Integer percentAdditionalTax) {
+        return price * (this.percentBaseTax + percentAdditionalTax) / 100;
+    }
+
+    public Boolean isTaxFree(Double price) {
+        if (price < 10) {
+            return true;
+        }
+        return false;
+    }
+}
+---- 
++
+In this example code, there are two methods:
+* `calculateTax`
++
+Receives the `price` and `percentAdditionalTax` parameters.
+* `isTaxFree`
++
+Receives the `price` parameter.
+
+=== Configure an HTTP Trigger
+
+. Click the *Listener* component on the canvas UI.
+. (Optional) Click the edit icon to change the name of the *Listener* component. 
+. (Optional) If you created a configuration file for the connector, select it, or add it to the XML:
```

Comment:
This isn't optional. If the configuration isn't set, the http operation won't work. So the first step should be to set up an http configuration (outside the flow).